lib/templateFactory.js

```diff
@@ -236,10 +236,14 @@ class TemplateFactory extends BaseFactory {
      * @param  {Object}   config.paginate         Pagination parameters
      * @param  {Number}   config.paginate.count   Number of items per page
      * @param  {Number}   config.paginate.page    Specific page of the set to return
+     * @param  {Number}   config.startTime        (Optional) Count only the builds that got started on or after startTime
+     * @param  {Number}   config.endTime          (Optional) Count only the builds that got ended on or before endTime
```

Comment:
can use brackets `[]` to indicate optional params
```suggestion
     * @param  {Number}   [config.startTime]        Count only the builds that got started on or after startTime
     * @param  {Number}   [config.endTime]          Count only the builds that got ended on or before endTime
```

Reply:
Done